### DISCUSSION TITLE
How can I see Vite logs from Quasar?

### DISCUSSION THREAD

**Discussion by gustavotoyota** [quasarframework/quasar#14267] [General - Components / Directives / etc]
I'm having problems with Vite and want to see what it's doing.
My JS files take ~2 ms to load, while my TS and Vue files take 10+ seconds.
I've been using Quasar CLI with Vite for some months and this problem now became unbearable.
I have to wait almost 2 minutes everytime I run` quasar dev -m ssr` to be able to develop.

My problem seems similar to this guy's: https://stackoverflow.com/questions/73204586/speed-up-vitejs-dev-mode-vue-3
But I'm not using WSL, so his solution didn't work for me.

Thanks.

**Comment by MilosPaunovic** [comment]
Is it the same when you create a fresh Quasar project?

**Reply by gustavotoyota** [reply]
It does look like there's something happening even on a fresh Quasar project, but I'm not sure.

On the first load MainLayout.vue takes 854ms to load:
![Screenshot (153)](https://user-images.githubusercontent.com/11639451/186897196-2e22acb8-e37f-4fca-bb43-ffc4dd075f85.png)

After restarting `quasar dev -m ssr`, EssentialLink.vue takes 400ms to load on top of MainLayout.vue's 593ms:
![Screenshot (154)](https://user-images.githubusercontent.com/11639451/186897281-9707d43b-9eaf-4bec-8dc7-fbe1c183d128.png)

This is what's happening in my project:
![Screenshot (157)](https://user-images.githubusercontent.com/11639451/186901548-323f4da7-50f0-4df3-b2f2-bdbec2ef9c77.png)
![Screenshot (158)](https://user-images.githubusercontent.com/11639451/186901830-cec18297-0123-4be0-87cd-bdd42fdcfb02.png)

**Reply by gustavotoyota** [reply]
Loading times do seem to be pretty irregular, the same files above that took long to load sometimes load relatively fast:
![Screenshot (161)](https://user-images.githubusercontent.com/11639451/186905132-8fba6465-2de1-4820-a14a-8902fe963707.png)
It's just slowness spread out everywhere.

**Comment by pdanpdan** [comment]
Are you on windows? Do you have the n+100 security agents mandated by the employer all started, all of them scanning any kind of file or process access?

**Reply by gustavotoyota** [reply]
I'm on Windows but I'm using my personal gaming computer for development.
I have no antivirus installed or anything heavy.
